docs/sources/reference/playlist.md

```diff
@@ -11,59 +11,95 @@ weight = 4
 
 # Playlist
 
-The Playlist is a special type of Dashboard that rotates through a list of Dashboards. A Playlist can be a great way to build situational awareness, or just show off your metrics to your team or visitors.
+The Playlist is a special type of Dashboard that rotates through a list of Dashboards. A Playlist can be a great way to build situational awareness, and show off your metrics to your team or visitors.
 
-Since Grafana automatically scales Dashboards to any resolution they're perfect for big screens!
+Grafana automatically scales Dashboards to any resolution, which makes them perfect for big screens.
+
+Access the Playlist feature from Grafana's side menu, in the Dashboard submenu.
```

Comment:
The submenu is actually called "Dashboard**s**". I think it sounds better without the `s`, but we should probably match the UI here. This should apply to all references in this document.

Reply:
Corrected these. Thanks!